packages/synthetic-chain/package.json

```diff
@@ -2,9 +2,7 @@
   "name": "@agoric/synthetic-chain",
   "version": "0.6.1",
   "description": "Utilities to build a chain and test proposals atop it",
-  "bin": {
-    "synthetic-chain": "dist/cli/cli.js"
-  },
+  "bin": "dist/cli/cli.js",
```

Comment:
Not sure if this was intended... I thought the former version was preferred.

Reply:
yeah, this must be Yarn's fault. The former version is what NPM does and I think we should stick with it. Oh well.